tests/regression/test_initing_causes_cert_deletion.py

```diff
@@ -0,0 +1,33 @@
+from nucypher.characters.lawful import Enrico
+from nucypher.config.storages import ForgetfulNodeStorage
+
+
+def test_initing_causes_cert_deletion(enacted_federated_policy):
```

Comment:
CircleCI config should be updated to include tests in `tests/regression`. Or perhaps even better, since this is a test for characters, move this test to `tests/characters/regression` (which I think it makes more sense anyway) and no need to update the CircleCI config.

Reply:
I think that the test can simply be removed, and a link to the historical commit added to #1554 

After all, as @derekpierre points out, the test at this point really just shows that python properly cleans up anonymous objects from the local namespace.

The alternative might be a test that shows the spookiness that occurs when you only create 5 or so anonymous objects this way, but that's not really relevant to the issue either.